samples/cellular/nrf_cloud_rest_cell_location/src/main.c

```diff
@@ -390,13 +390,31 @@ static void do_location_card_enable(void)
 		.ui = &ui
 	};
 
+	struct nrf_cloud_modem_info mdm_inf = {
+		/* Include all available modem info. Change to NRF_CLOUD_INFO_NO_CHANGE to
+		 * reduce the volume of data transmitted.
+		 */
+		.device = NRF_CLOUD_INFO_SET,
+		.network = NRF_CLOUD_INFO_NO_CHANGE,
+		.sim = NRF_CLOUD_INFO_NO_CHANGE,
+		/* Use the modem info already obtained */
+		.mpi = &mdm_param,
+		/* Include the application version */
+		.application_version = CONFIG_REST_CELL_LOCATION_SAMPLE_VERSION
+	};
+
+	struct nrf_cloud_device_status dev_status = {
+		.modem = &mdm_inf,
+		.svc = &svc_inf
+	};
+
 	/* Keep the connection alive so it can be used by the initial cellular
 	 * positioning request.
 	 */
 	rest_ctx.keep_alive = true;
 
 	LOG_INF("Enabling location card...");
-	err = nrf_cloud_rest_shadow_service_info_update(&rest_ctx, device_id, &svc_inf);
+	err = nrf_cloud_rest_shadow_device_status_update(&rest_ctx, device_id, &dev_status);
```

Comment:
This commit mentions that the sample is now reporting whether we are using LTE or Wi-Fi, but I don't see a change here that would do that

If the sample reporting that is just a side-effect of another commit, then consider not mentioning it in the commit message here

Reply:
It's a side effect.

Reply:
The commit message already clarifies this:
`The latter information is done automatically when using nrf_cloud_rest_shadow_device_status_update().`